### ISSUE TITLE
Question about the reported results.

### ISSUE THREAD

**Issue by ha-lins** [kavehhassani/mvgrl#1]
Hi @kavehhassani ,

Thanks for the released code! I have a minor question about the reported results. When I ran the code, some results are printed and I wonder how to obtain the reported results in the paper (e.g. 89.7 for MUTAG dataset in graph classification). Is it maybe just to pick out the highest result? Thanks a lot!

Here are some of results printed:
`####################MUTAG####################
Dataset: MUTAG, Layer:2, Batch: 32, Epoch: 20, Seed: 123
0.8880701754385966 0.0732108857685375
Dataset: MUTAG, Layer:2, Batch: 32, Epoch: 20, Seed: 132
0.8919590643274855 0.07107527976346202
Dataset: MUTAG, Layer:2, Batch: 32, Epoch: 20, Seed: 321
0.898684210526316 0.06647971246853003
Dataset: MUTAG, Layer:2, Batch: 32, Epoch: 20, Seed: 312
0.8581578947368422 0.07014429742760363
Dataset: MUTAG, Layer:2, Batch: 32, Epoch: 20, Seed: 231
0.8964327485380117 0.08969212337006427
Dataset: MUTAG, Layer:2, Batch: 32, Epoch: 40, Seed: 123
0.8545029239766082 0.08952242299027656
`

**Comment by kavehhassani** [comment]
Following https://openreview.net/forum?id=r1lfF2NYvH we report the mean of the best config.

**Comment by ha-lins** [comment]
Thanks for your reply! @kavehhassani 

In my experiments of the graph classification, the mean results of 5 runs of the best config seem to be lower slightly than the ones reported in the paper. Here are some of what I obtained for the graph classification task:

`                     My experiments       In the paper`
`MUTAG:       89.27 ±  0.77            89.7 ± 1.1` 
`PTC-MR:       58.3   ±  0.64            62.5 ± 1.7`
`IMDB-BIN:   73.8   ±  0.91            74.2 ± 0.7`  

It seems to have utilized all possible hyperparameters in the code of graph/train.py. So could you give me some advice for reproducibility? Thanks!
